packages/charting/src/components/BenchmarkBarChart/BenchmarkBarChart.base.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+
+import { mergeStyles, FocusZone } from 'office-ui-fabric-react';
+import { IProcessedStyleSet, DefaultPalette, IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import { IBenchmarkBarChartProps, IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle } from './BenchmarkBarChart.types';
+
+const getClassNames = classNamesFunction<IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle>();
+
+export class BenchmarkBarChartBase extends React.Component<IBenchmarkBarChartProps> {
+  private _classNames: IProcessedStyleSet<IBenchmarkBarChartStyle>;
+
+  constructor(props: IBenchmarkBarChartProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const { chartTitle, benchmarkLabel, data, benchmarkData, totalData } = this.props.data;
+
+    const dataRatio = Math.round((data / totalData) * 100);
+    const benchmarkRatio = Math.round(((benchmarkData ? benchmarkData : 0) / totalData) * 100);
+
+    let percentageContent = data + '/' + totalData;
+    if (this.props.isPercentage) {
+      percentageContent = dataRatio + '%';
+    }
+
+    let benchmarkAriaLabel = benchmarkLabel || '';
+    benchmarkAriaLabel += benchmarkData;
+
+    const benchmarkStyles: IStyle = {
+      marginLeft: 'calc(' + benchmarkRatio + '% - 8px)'
+    };
+
+    const barChartColor = this.props.color || DefaultPalette.purple;
```

Comment:
Should we just move this to styles? Feels kind of weird to have color as a direct prop

Reply:
The color value is used to fill the rect. It is not used in styles. So I think it shouldn't be moved to styles.

Reply:
how does this color work in high contrast mode? Have you run an accessibility test pass or talked to accessibility team?

---
In reply to: [283711599](https://github.com/OfficeDev/office-ui-fabric-react/pull/9039#discussion_r283711599) [](ancestors = 283711599)

Reply:
I don't know what the standard is for color working in high contrast mode. Could you give me an example about color working in high contrast mode on existing charting components? Besides, Does the fabric project have tools to run an accessibility test? I have no idea how to run an accessibility test. It is helpful if you provide us the accessibility team's contact information. Thanks a lot.

Reply:
You can run accessibility insights https://accessibilityinsights.io/docs/en/web/overview

Reply:
I've run an accessibility test with the Accessibility Insights for Web extension. It doesn't detect any   high contrast mode accessibility issue about the benchmarkBarChart component.

![image](https://user-images.githubusercontent.com/12230173/57893073-17106280-7874-11e9-9ecc-7822f19a83d9.png)